src/questions/intro-to-lending-borrowing.yaml

```
@@ -1,19 +1,404 @@
-- uuid: 3c35c4d1-4b6d-4071-9019-070426f5bf42
+- uuid: 68d9898f-4838-4218-9356-dba391a63fdd
   type: SingleChoice
-  content: What is the main purpose of DeFi?
-  hint: Purpose of DeFi in the current scenario
-  explanation: Decentralised Finance is responsible for providing financial solutions for existing problems in the centralised system using blockchain.
+  content: What is collateral and over collateralization?
+  hint: Collateral and over-collateralization
+  explanation: Collateral is a security locked by borrower for borrowing. Overcollateralization is a process in which collateral is locked at higher value than borrowed value in order to cover the potential losses of the collateral.
+  answerKeys:
+    - C
+  subTopics:
+    - why-do-we-lending-borrowing
+  difficultyLevel: Low
+  choices:
+    - content: Collateral is returns on lent assets.
+      key: A
+    - content: Collateral net value of borrowed amount and Overcollateralization is an investment strategy for increasing the potential return of investment.
+      key: B
+    - content: Collateral is a security locked by borrower and locking collateral at a higher value than the borrowed amount is called over-collateralization.
+      key: C
+    - content: None of these
+      key: D
+
+- uuid: 9b09ff86-e6d3-4861-987d-aa819fe5622a
+  type: MultipleChoice
+  content: What are the use cases of lending and borrowing?
+  hint: To gain interest and get money in urgent situations when capital is insufficient.
+  explanation: To gain interest and borrowing helps you to get money in urgent situations or for improvement purposes when capital is insufficient.
```

```suggestion
  explanation: Lending helps in .... and borrowing helps you to get money in urgent situations or for improvement purposes when capital is insufficient.
```
Explain Lending clearly in a separate sentence